vendor/github.com/docker/docker/pkg/jsonmessage/jsonmessage.go

```diff
@@ -48,10 +48,10 @@ func (p *JSONProgress) String() string {
 		timeLeftBox string
 	)
 
-	ws, err := term.GetWinsize(p.terminalFd)
-	if err == nil {
-		width = int(ws.Width)
-	}
+	//	ws, err := term.GetWinsize(p.terminalFd)
```

Comment:
the rest of this looks safe, but how do you know this terminal width thing is not needed?